sdk/tables/azure-data-tables/tests/test_table_client.py

```diff
@@ -253,8 +252,76 @@ def test_client_with_sas_token(self, tables_storage_account_name, tables_primary
             )
             for e in entities:
                 pass
+
+        with pytest.raises(ValueError) as ex:
+            client = TableClient.from_table_url(sas_url, credential=AzureSasCredential(sas_token))
```

Comment:
We're only doing this check using `from_table_url`?
We can also do this check using the `__init__()` constructor and the `from_connection_string` constructor?

Reply:
Also with TableServiceClient